### PR TITLE
Update virctl commands

### PR DESCRIPTION
**Pull request by apinnick**
Changes: 
- Moved virtctl installation to same assembly as virtctl commands.
- Added some missing commands.
- Standardized virtctl examples.
- Updated libguestfs tools sections to make them clearer.

Version(s): 13+

Issue: None

Link to docs preview: [Using the virtctl and libguestfs CLI tools](https://61571--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-using-the-cli-tools.html#virt-virtctl-commands_virt-using-the-cli-tools)

QE review: NA
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by ousleyp — COMMENTED** [review]
Not a full peer review, per se, but it's looking really good to me! Just one concern with the title change and a pre-existing nit to fix. Thanks for doing this!

File: modules/virt-installing-virtctl-client-yum.adoc [hunk 1]
```diff
@@ -4,16 +4,32 @@
...
-= Installing the virtctl client using the yum utility
+= Installing virtctl on {op-system-base}
 
-Install the `virtctl` client from the `kubevirt-virtctl` package.
+You install the `virtctl` RPM on {op-system-base-full} by enabling the {VirtProductName} repository and installing the `kubevirt-virtctl` package.
 
 .Prerequisites
 
-* You enabled an {VirtProductName} repository on your {op-system-base-full} system.
+* Your system is registered to a Red Hat account with an active subscription to the "Red Hat Container Native Virtualization" entitlement.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply modules/virt-installing-virtctl-client-yum.adoc:13]: There must be a better way to phrase this. I will check. I copied this from one oof the deleted files.

File: _topic_maps/_topic_map.yml [hunk 2]
```diff
@@ -3648,7 +3646,7 @@ Topics:
   Distros: openshift-origin
 - Name: Security policies
   File: virt-security-policies
-- Name: Using the CLI tools
+- Name: Using virtctl and libguestfs
```
[the remark this answers is not in the document]
Comment (apinnick) [reply _topic_maps/_topic_map.yml:3649]: I am concerned that the title would be very long in the nav tree. I will think about it.

[the remark this answers is not in the document]
Comment (apinnick) [reply _topic_maps/_topic_map.yml]: I guess it looks OK. Changed it.

File: virt/virt-using-the-cli-tools.adoc [hunk 3]
```diff
@@ -1,34 +1,32 @@
...
 :toclevels: 3
 
 toc::[]
 
-The two primary CLI tools used for managing resources in the cluster are:
+You can manage {VirtProductName} resources by using the `virtctl` command line tool.
 
-* The {VirtProductName} `virtctl` client
-* The {product-title} `oc` client
+You can also deploy a `libguestfs-tools` container by using `virtctl`. link:https://libguestfs.org[`Libguestfs`] is a set of tools for accessing and modifying virtual machine (VM) disk images.
```
Comment (GroceryBoyJr) [inline virt/virt-using-the-cli-tools.adoc:12]:
```suggestion
You can also deploy a `libguestfs-tools` container by using `virtctl`. link:https://libguestfs.org[`libguestfs`] is a set of tools for accessing and modifying virtual machine (VM) disk images.
```

**Review by GroceryBoyJr — COMMENTED** [review]
@apinnick only one tiny nit. Fantastic work!

**Pull request merged**